src/passport-saml/saml-post-signing.ts

```diff
@@ -15,15 +16,15 @@ interface SignSamlPostOptions {
 export function signSamlPost(samlMessage: string, xpath: string, options: SignSamlPostOptions) {
   if (!samlMessage) throw new Error('samlMessage is required');
   if (!xpath) throw new Error('xpath is required');
-  if (!options || !options.privateCert) throw new Error('options.privateCert is required');
+  if (!options || (!options.privateCert && !options.privateKey)) throw new Error('options.privateCert or options.privateKey is required');
```

Comment:
What do you think of issuing a deprecation warning if someone is using `privateCert`? This way, we can remove this extra code with a future semver-major release.

Reply:
@cjbarth That's OK if you want to add the warning.